### PR TITLE
Add `fetch-on-empty` optional attribute

### PR DESCRIPTION
**Pull request by loisolire**
This attribute, when set, allows the auto-complete-element to fetch results with empty query value.

### CODE REVIEW

**Comment by primer-css** [comment]
:wave: Hello and thanks for pinging us! This issue or PR has been added to our inbox and a Design Infrastructure first responder will review it soon.
- :art: **If this is a PR that includes a visual change**, please make sure to add screenshots in the description or deploy this code to a lab machine with instructions for how to test.
- :fast_forward: **If this is a PR that includes changes to an interaction**, please include a [video recording](https://support.apple.com/en-us/HT208721) in the description.
- :warning: **If this is urgent**, please visit us in [#primer](https://github.slack.com/archives/CSGAVNZ19) on Slack and tag the first responders listed in the channel topic.

**Review by keithamus — APPROVED** [review]
Seems like a good addition :+1:

**Comment by MaximeRnR** [comment]
Do we have a idea for when it will be merged ^^ ?

File: src/auto-complete-element.ts [hunk 1]
```diff
@@ -55,6 +55,18 @@ export default class AutocompleteElement extends HTMLElement {
...
+  get fetchOnEmpty(): boolean {
+    return this.hasAttribute('fetch-on-empty')
+  }
+
+  set fetchOnEmpty(fetchOnEmpty: boolean) {
+    if (fetchOnEmpty) {
+      this.setAttribute('fetch-on-empty', '')
+    } else {
+      this.removeAttribute('fetch-on-empty')
+    }
```
Comment (keithamus) [inline src/auto-complete-element.ts:67]:
```suggestion
    this.toggleAttribute('fetch-on-empty', fetchOnEmpty)
```

**Comment by MaximeRnR** [comment]
Hello, do we have any idea when this will be merged ? Thanks a lot for your time !

**Review by keithamus — APPROVED** [review]

**Comment by keithamus** [comment]
Apologies, this should have been handled by the first responder staff member in rotation on January.

**Comment by keithamus** [comment]
This has been released as `v3.4.0`. Thanks for your efforts @loisolire! Thanks for bumping this @MaximeRnR

**Pull request merged**